modules/nf-core/bwamem2/mem/main.nf

```diff
@@ -1,25 +1,24 @@
 process BWAMEM2_MEM {
-    tag "$meta.id"
+    tag "${meta.id}"
     label 'process_high'
-
     conda "${moduleDir}/environment.yml"
-    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
-        'https://depot.galaxyproject.org/singularity/mulled-v2-e5d375990341c5aef3c9aff74f96f66f65375ef6:2d15960ccea84e249a150b7f5d4db3a42fc2d6c3-0' :
-        'biocontainers/mulled-v2-e5d375990341c5aef3c9aff74f96f66f65375ef6:2d15960ccea84e249a150b7f5d4db3a42fc2d6c3-0' }"
+    container "${workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container
+        ? 'https://depot.galaxyproject.org/singularity/mulled-v2-e5d375990341c5aef3c9aff74f96f66f65375ef6:2d15960ccea84e249a150b7f5d4db3a42fc2d6c3-0'
+        : 'biocontainers/mulled-v2-e5d375990341c5aef3c9aff74f96f66f65375ef6:2d15960ccea84e249a150b7f5d4db3a42fc2d6c3-0'}"
 
     input:
     tuple val(meta), path(reads)
     tuple val(meta2), path(index)
     tuple val(meta3), path(fasta)
-    val   sort_bam
+    val sort_bam
 
     output:
-    tuple val(meta), path("*.sam")  , emit: sam , optional:true
-    tuple val(meta), path("*.bam")  , emit: bam , optional:true
-    tuple val(meta), path("*.cram") , emit: cram, optional:true
-    tuple val(meta), path("*.crai") , emit: crai, optional:true
-    tuple val(meta), path("*.csi")  , emit: csi , optional:true
-    path  "versions.yml"            , emit: versions
+    tuple val(meta), path("*.sam"), emit: sam, optional: true
+    tuple val(meta), path("*.bam"), emit: bam, optional: true
+    tuple val(meta), path("*.cram"), emit: cram, optional: true
+    tuple val(meta), path("*.crai"), emit: crai, optional: true
+    tuple val(meta), path("*.csi"), emit: csi, optional: true
```

Comment:
Why has this gone all wibbly-wobbly?

Reply:
You need to turn on the Harshil Alignment in the settings. I'll make a PR for the workspace.